src/js/apps/patients/worklist/filters_app.js

```diff
@@ -5,15 +5,17 @@ import intl from 'js/i18n';
 import App from 'js/base/app';
 
 import OwnerDroplist from 'js/views/patients/shared/components/owner_component';
-import { FiltersView, GroupsDropList, ClinicianDropList, TypeToggleView, RoleComponent } from 'js/views/patients/worklist/filters_views';
+import { FiltersView, GroupsDropList, ClinicianDropList, TypeToggleView, RoleComponent, DateFilterView } from 'js/views/patients/worklist/filters_views';
 
 export default App.extend({
   stateEvents: {
     'change:clinicianId': 'showFilters',
+    'change:date change:relativeDate change:month': 'showDateFilterView',
```

Comment:
I think this entire stateEvents should probably be removed.  I don't think the `change:clinicianId` is doing anything.